stdlib/float.mli

```diff
@@ -217,6 +254,20 @@ external floor : float -> float = "caml_floor_float" "floor"
     equal to [f].
     The result is returned as a float. *)
 
+external nextafter : float -> float -> float
```

Comment:
I would also like to understand where you got that signature/name from and if you really use it will all the properties it has, it's semantics is quite complex.

FWIW IEEE-754 simply has `nextUp` and `nextDown` and in `Gg.Float` I expressed these as [`pred` and `succ`](https://github.com/dbuenzli/gg/blob/4d4c214f9c8ad8790a6dcca7f6fcb3c263dd16e1/src/gg.mli#L181-L192) to follow more traditional OCaml names.

If kept as such I would still suggest `next_after` rather than `nextafter`.

Reply:
`nextafter` is a C99 function/POSIX function, and the signature here is conformant. I'm not sure of why it was deemed useful in C, though.

Reply:
@yakobowski Thanks for the input. Given what you say I retract the suggestion of calling it `next_after`.

Reply:
@dbuenzli We have to be careful to maintain a uniform style because the same argument applies to `copysign` and `signbit`.  I'll have to check whether standard names were also added an underscore in the standard library.

Reply:
Also I note that Julia which you cited doesn't [seem](https://docs.julialang.org/en/v1.0/base/numbers/#Base.float) have that function but `nextfloat`, `prevfloat`. I would personally rather be in favour of having these ones under the `pred` and `succ` (for `IntXX` module uniformity) rather than this function.

Reply:
`nextafter` is available on virtually any platform, including Windows.   [Julia](http://www.julialang.org/) implements it in its [openlibm](https://github.com/JuliaMath/openlibm/blob/master/src/s_nextafter.c) project.  

`succ` and `prev` are easily defined in terms of `next_after`:
```ocaml
let succ x = next_after x infinity
let pred x = next_after x neg_infinity
```
and I can include them in `Float` if desired.